recipe/meta.yaml

```
@@ -31,20 +29,28 @@ requirements:
 test:
   imports:
     - dotenv
+  source_files:
+    - tests
   requires:
     - pip
+    - pytest
+    - sh   # [not win]
```

```suggestion
    - pytest >=3.9
    - sh >2   # [not win]
```

Do these tests not also require the rest of the dependencies in their [requirements.txt](https://github.com/theskumar/python-dotenv/blob/v1.1.0/requirements.txt)? Maybe for the ones we're skipping?

---

hmmm, It doesnt seem so? I think the upstream and the click requirement is the bare minium required for this package but it also wanted to make it a run_constrained. When building, it doesn't seem to prompt for any of those dependencies, do you think it might be worth a  try to add them to run constrained perhaps?

---

No, if they aren't anything missing in those tests we're skipping then they aren't needed. Several of the ones we don't include are things like code formatters and coverage reporters so I figured they weren't all necessary. I _did_ think ipython would be needed since it looks like they use it in a few places but maybe they try/except it out.

---

Ah gotcha, I can add ` ipython` under run_constrained just for safe measure!

---

Well, there's no constraint on it so it won't matter